frontend/app/features/login/styles/login.module.css

```diff
@@ -30,4 +30,10 @@
   .button {
     width: 100%;
   }
+
+  .disabledLink {
+    color: gray;
+    cursor: not-allowed;
+    text-decoration: none;
+  }
   
```

Comment:
Please check all of your files, and add new line at the end if there is none to resolve this
![image](https://github.com/user-attachments/assets/6fcb6902-ac6f-4e4f-8116-9a3e287ee3aa)